arraycontext/transform_metadata.py

```diff
@@ -53,6 +53,24 @@ class ElementwiseMapKernelTag(Tag):
         in a finite-element discretization.
     """
 
+
+class ParameterValue(UniqueTag):
+    """A tag that applies to :class:`loopy.ValueArg`. Instances of this tag
+    are initialized with the value of the parameter and may be later used
+    to fix value of the parameter.
+    """
+
+    def __init__(self, value):
+        self.value = value
+
+
+class IsDOFArray(Tag):
```

Comment:
`DOFArray`s can't currently be tagged. (They're containers.) Do you mean that they're group arrays?

Reply:
Clarified this applies to loopy `ArrayArg` instances.